src/main/java/com/neva/felix/webconsole/plugins/search/core/OsgiExplorer.java

```diff
@@ -61,28 +62,50 @@ public OsgiExplorer(BundleContext context) {
 	}
 
 	public File findDir(Long bundleId) {
-		return new File(String.format(BUNDLE_PATH_FORMAT, context.getProperty(BUNDLE_STORAGE_PROP), bundleId));
+		String bundleStorage = context.getProperty(BUNDLE_STORAGE_PROP);
```

Comment:
If you used default config of clean Felix.. This must be covered. I will look at it today